models/ed/R/met2model.ED2.R

```diff
@@ -54,12 +55,18 @@ met2model.ED2 <- function(in.path, in.prefix, outfolder, start_date, end_date, l
   dl <- c(0, 32, 61, 92, 122, 153, 183, 214, 245, 275, 306, 336, 367)
   month <- c("JAN", "FEB", "MAR", "APR", "MAY", "JUN", "JUL", "AUG", "SEP", "OCT", "NOV", "DEC")
   mon_num <- c("01", "02", "03", "04", "05", "06", "07", "08", "09", "10", "11", "12")
-  day2mo <- function(year, day) {
-    leap      <- lubridate::leap_year(year)
-    mo        <- rep(NA, length(day))
-    mo[leap]  <- findInterval(day[leap], dl)
-    mo[!leap] <- findInterval(day[!leap], dm)
-    return(mo)
+  day2mo <- function(year, day, leap_year_agnostic) {
```

Comment:
You have `leap_year_agnostic` here, but use `leap_year` in the function. Assuming this is a typo?

Reply:
Also, since you are modifying the function here, you need to change its call later in the script ([line 177](https://github.com/PecanProject/pecan/blob/8f0e23dd2a25568fd62effec78d8c32ace071c78/models/ed/R/met2model.ED2.R#L177)).